.github/workflows/dart.yml

```
@@ -0,0 +1,73 @@
+name: Dart
```

Juts saw the change on the flutter CI now, reverting:

```suggestion
name: sentry-dart
```